.gitignore

```diff
@@ -1,5 +1,5 @@
 .DS_Store
 node_modules/
 .env
-config.js
+config
```

Comment:
My config file is config.js so I need that line. Can you update this before we merge? 😅

Reply:
My bad! Fixing now.